examples/backpack-demo/index.js

```diff
@@ -7,22 +7,204 @@
 /**
  * @fileoverview Backpack demo initialization.
  */
+var toolbox = {
+  "kind": "categoryToolbox",
```

Comment:
Can you change this file to use single quotes?

Reply:
done

Reply:
done